vendor.go

```diff
@@ -109,24 +110,22 @@ func getGoPath() string {
 	if gopath != "" {
 		return gopath
 	}
-	home := os.Getenv("HOME")
-	if home == "" {
-		fmt.Println("no GOPATH env var found and no HOME to infer GOPATH from")
-		os.Exit(1)
-	}
-	return filepath.Join(home, "go")
+	u, err := user.Current()
+	abortonerr(err, "getting current user")
+	return filepath.Join(u.HomeDir, "go")
 }
 
-func vendorPackage(gohome string, rootdir string, pkg string) {
-	srcpkgpath := filepath.Join(gohome, "src", pkg)
+func createDir(dir string) {
+	err := os.MkdirAll(dir, 0774)
+	abortonerr(err, fmt.Sprintf("creating dir[%s]", dir))
+}
 
 func vendorPackages(depsGoHome string, projectdir string) {
-	depsrootdir := path.Join(depsGoHome, "src")
-	projectVendorPath := path.Join(projectdir, "vendor")
+	depsrootdir := filepath.Join(depsGoHome, "src")
```

Comment:
Can't find a difference between path and filepath, path always uses "/" and filepath is portable ?

Reply:
yep